cypress/features/visual/grid.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Grid component
+  I want to check that all examples of Grid component render correctly
+
+  @positive
+  @applitools
+  Scenario: Check Grid has "<title>" as title
```

Comment:
Do we need `<"title">` in this form? It's normal `scenario` not `scenario outline`

Reply:
updated